### ISSUE TITLE
[IFRAMES] has_selector? and wait_for_selector? dont seem to work for iframes

### ISSUE THREAD

**Issue by lesliepc16** [natritmeyer/site_prism#66]
The helper functions added by site prism do not appear to work for iFrames.

For example, say I have the following section:

``` rb
class MySection < SitePrism::Section
  class PreviewIframe < SitePrism::Page
  end
  iframe :preview, PreviewIframe, "#preview"
end
```

This is returning false even if the preview is displayed on the page.

``` rb
my_section.has_preview?
```

wait_for_preview doesn't appear to work either.

I verified that the iframe works in that I can do actions within it. It's just the wait_for_preview and has_preview? methods that aren't behaving.

Is this as designed? It'd be great to have these working for iframes also. Thanks!

**Comment by natritmeyer** [comment]
Sorry, only just noticed this. Still a problem?

**Comment by luke-hill** [comment]
Hi there. Just a quick bump. Could you provide an SSCCE where this is falling over so we can triage this.

Thanks

**Comment by luke-hill** [comment]
So I've re-created this and compared to the unit / feature tests we have and think I've spotted the problem (Although why some of the helpers work I'm not sure). I managed to get a similar example passing.

You're code example has the references defined incorrectly. You've namespaced your iFrame inside your other class, and as such need to reference the ClassName accordingly.

It could be something that's causing some of the helpers to work and others not. But essentially it wouldn't be looked into unless you've got a valid SSCCE

Reference Cuke that calls both of your failing methods: https://github.com/natritmeyer/site_prism/blob/master/features/step_definitions/iframe_steps.rb#L4-L6
